lang/java/mapred/src/main/java/org/apache/avro/mapreduce/AvroOutputFormatBase.java

```diff
@@ -85,6 +86,15 @@ protected static CodecFactory getCompressionCodec(TaskAttemptContext context) {
     return CodecFactory.nullCodec();
   }
 
+  private Path getWorkPathFromCommitter(TaskAttemptContext context) throws IOException {
+    try {
+      OutputCommitter committer = getOutputCommitter(context);
+      return (Path) committer.getClass().getMethod("getWorkPath").invoke(committer);
+    } catch (ReflectiveOperationException e) {
+      throw new AvroRuntimeException("Committer does not have method getWorkPath", e);
```

Comment:
might be nice to include info on the committer (e.g. its toString() value, or at least classname) here

Reply:
Add in https://github.com/apache/avro/pull/1618/commits/095d4b4e7af33054ffead0257adef3e20e2e2d55 .